postgres-appliance/scripts/configure_spilo.py

```diff
@@ -107,10 +108,21 @@ def write_certificates(environment, overwrite):
             '-out',
             environment['SSL_CERTIFICATE_FILE'],
         ]
-        logging.info('Generating ssl certificate')
+        logging.info('Autogenerating ssl certificate')
         p = subprocess.Popen(openssl_cmd, shell=False, stdout=subprocess.PIPE, stderr=subprocess.STDOUT)
         output, _ = p.communicate()
         logging.debug(output)
+        logging.info('Generating ssl certificate')
```

Comment:
Redundant message, duplicates https://github.com/zalando/spilo/pull/624/files#diff-e4dcf964214d58d733ae9874d4a2ab4b8e9eb99213307e1666af5dad21a326bbR111
```suggestion
```

Reply:
Is it OK for you if I remove it?